tests/gpu_benchmark/modal_memory_hard.py

```
@@ -0,0 +1,356 @@
+RUST
+
+# Compile and run with memory measurement
+echo "Compiling..."
+source $HOME/.cargo/env
+rustc mempow.rs -O -o mempow
+
+echo "Running benchmark..."
+time ./mempow
+"""
+    
+    result = subprocess.run(
```

Ruff (0.14.14)

`subprocess` call: check for execution of untrusted input

(S603)